packages/block-editor/src/components/block-tools/index.js

```diff
@@ -20,6 +20,17 @@ import { store as blockEditorStore } from '../../store';
 import BlockContextualToolbar from './block-contextual-toolbar';
 import { usePopoverScroll } from './use-popover-scroll';
 
+/**
+ * Returns true if the container contains the document active element.
+ *
+ * @param {HTMLElement} container An HTML element.
+ *
+ * @return {boolean} Whether the container contains the currently document active element.
```

Comment:
Tiny nit to clarify the wording, feel free to skip it!

```suggestion
 * @return {boolean} Whether the container contains the currently active element in the document.
```